docs/modules/publisher.md

```diff
@@ -118,8 +118,7 @@ The simplest way to configure OPC Publisher is via a configuration file. A basic
 ]
 ```
 
-Another example configuration files are provided via [`publishednodes_2.5.json`](publishednodes_2.5.json?raw=1)
-and [`publishednodes_2.8.json`](publishednodes_2.8.json?raw=1) files.
+Another example configuration files are provided via [`publishednodes_2.5.json`](publishednodes_2.5.json?raw=1) and [`publishednodes_2.8.json`](publishednodes_2.8.json?raw=1).
```

Comment:
Nit: Other instead of Another, what do you think?

Reply:
agree! :)